pkg/kubernetes/kube.go

```diff
@@ -866,3 +895,116 @@ func (kc *Client) PersistentVolExists(volName, expectedPhase string) error {
 	}
 	return nil
 }
+
+func (kc *Client) VerifyInstanceGroups() error {
+	igs, err := kc.ListInstanceGroups()
+	if err != nil {
+		return err
+	}
+
+	for _, ig := range igs.Items {
+		if !isInstanceGroupStatus(&ig, "ready") {
+			return errors.New("BDD >> failed while instanceGroup state")
```

Comment:
Should we change this to something else? I realize this is how it was internally, but error is a bit confusing.

Reply:
improved the error messaging